models/MLmodels/ModelsRunner.py

```diff
@@ -0,0 +1,51 @@
+from MLmodels.DataReader import DataSample
+from MLmodels.DecisionTree import DecisionTree
+from MLmodels.ElasticNets import ElasticNet
+from MLmodels.Lasso import Lasso
+from MLmodels.NeuralNetwork import NeuralNetwork
+from MLmodels.RandomForest import RandomForest
+from MLmodels.Ridge import Ridge
+from visualizers.models_visual_training_comparator import Models_comparator
+import warnings
+
+
+class ModelsRunner:
+
+    data = DataSample()
+
+    models_dict = {
+        'Lasso': Lasso(data), 'Ridge': Ridge(data), 'ElasticNet': ElasticNet(data),
+        'DecisionTree': DecisionTree(data), 'RandomForest': RandomForest(data),
+        'NeuralNetwork': NeuralNetwork(data)
+    }
+
+    def __init__(self):
+        pass
```

Comment:
Skapa en parameter för att skicka in data som inte är DataSample, likt vad som är gjort på modellerna.